src/Illuminate/Support/HtmlString.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Illuminate\Support;
+
+use Illuminate\Contracts\Support\Htmlable;
+
+class HtmlString implements Htmlable
```

Comment:
I thought about submitting without this concrete class--and I'm happy to remove it if that will help getting this merged--but I included it to provide a simple, out-of-the-box userland helper akin to the "Safe" HTML helpers found in other templating languages.

```
return view('home', ['button' => new HtmlString('<a href="#">Button</a>')]);
```